pyccel/parser/semantic.py

```diff
@@ -814,8 +839,17 @@ def _handle_function(self, expr, func, args, **settings):
         =======
         new_expr : FunctionCall or PyccelInternalFunction
         """
+        if isinstance(func, FunctionDef) and 'kernel' in func.decorators:
+            errors.report(MISSING_KERNEL_CONFIGURATION, symbol = expr, severity = 'fatal')
```

Comment:
@EmilyBourne  not sure if this is the correct place of raising these errors

Reply:
If this is where you detect it then it seems fine to me. Where else would you want to raise it?